extensions/amp-form/0.1/amp-form.js

```diff
@@ -810,9 +810,9 @@ export class AmpForm {
 
   /**
    * Transition the form to the submit success state.
-   * @param {!JsonObject|string|undefined} response
+   * @param {!JsonObject} response
    * @param {!FetchRequestDef} request
-   * @return {!Promise}
+   * @return {!Promise<string>}
```

Comment:
This type is incorrect. Doesn't `gulp check-types` complain about this?

Reply:
it does not complain about it. removed. Will fix in the next PR along with your other comments.